app/src/main/java/com/tokopedia/sample/utils/UserPrefs.kt

```diff
@@ -0,0 +1,35 @@
+package com.tokopedia.sample.utils
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.SharedPreferences
+/**
+ * Author errysuprayogi on 10,January,2020
+ */
+
+//@SuppressLint("StaticFieldLeak")
+class UserPrefs private constructor(private val context: Context) {
+    private val settings: SharedPreferences
+    fun saveString(key: String?, value: String?) {
+        settings.edit().putString(key, value).commit()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/TinkerSampleApp/blob/da7c579084553b6a2bf1a7a0ec5611865023204a/app/src/main/java/com/tokopedia/sample/utils/UserPrefs.kt#L14"></span>Consider using <code>apply()</code> instead; <code>commit</code> writes its data to persistent storage immediately, whereas <code>apply</code> will handle it in the background</td>
    </tr>
  </tbody>
</table>